src/main/webapp/app/overview/courses.component.html

```diff
@@ -25,9 +25,12 @@ <h4 class="ms-2 fw-medium">{{ nextRelevantExam.title }}</h4>
     <div class="col-sm-8">
         <h3 class="fw-medium" jhiTranslate="artemisApp.studentDashboard.title"></h3>
     </div>
-    <div class="col-sm-auto d-flex">
-        <a class="btn btn-primary" [routerLink]="['/courses/enroll']">{{ 'artemisApp.studentDashboard.enroll.title' | artemisTranslate }}</a>
-    </div>
+
+    @if (regularCourses.length && recentlyAccessedCourses.length) {
```

Comment:
What happens in case the student is already enrolled in a course but has no recently accessed ones?
I think we still need to show the enrol button in the top-right corner in that case.
```suggestion
    @if (regularCourses.length) {
```

Reply:
True, fixed it with latest commit